web/components/feed/feed-answer-comment-group.tsx

```diff
@@ -28,23 +31,24 @@ export function FeedAnswerCommentGroup(props: {
   items: ActivityItem[]
   type: string
   betsByCurrentUser?: Bet[]
-  comments?: Comment[]
+  commentsByCurrentUser?: Comment[]
 }) {
-  const { answer, items, contract, betsByCurrentUser, comments } = props
+  const { answer, items, contract, betsByCurrentUser, commentsByCurrentUser } =
+    props
   const { username, avatarUrl, name, text } = answer
   const answerElementId = `answer-${answer.id}`
   const user = useUser()
   const mostRecentCommentableBet = getMostRecentCommentableBet(
     betsByCurrentUser ?? [],
-    comments ?? [],
+    commentsByCurrentUser ?? [],
     user,
     answer.number + ''
   )
   const prob = getDpmOutcomeProbability(contract.totalShares, answer.id)
   const probPercent = formatPercent(prob)
   const [open, setOpen] = useState(false)
   const [showReply, setShowReply] = useState(false)
-  const isFreeResponseContractPage = comments
+  const isFreeResponseContractPage = commentsByCurrentUser
```

Comment:
nit: If the variable starts with `is` one might assume it is a boolean. You could assign to `!!commentsByCurrentUser` to  convert it.